### ISSUE TITLE
jasmine/no-focused-tests' endLine includes too much of the test

### ISSUE THREAD

**Issue by DamienCassou** [tlvince/eslint-plugin-jasmine#230]
**Describe the bug**

`jasmine/no-focused-tests` returns an `endLine` value that represents the whole test. This makes it particularly inconvenient to code on a `fit` test. As you can see, [everything is underlined](https://github.com/flycheck/flycheck/issues/1730):

![2020-04-12-161219](https://user-images.githubusercontent.com/217543/79070829-68b9ab00-7cd8-11ea-884e-e09798d24678.png)

```json
[
  {
    "filePath": "…",
    "messages": [
      {
        "ruleId": "jasmine/no-focused-tests",
        "severity": 2,
        "message": "Unexpected fit.",
        "line": 2,
        "column": 2,
        "nodeType": "CallExpression",
        "endLine": 9,
        "endColumn": 4
      }
    ],
    "..."
  }
]
```

**To Reproduce**

1. Write this code:

    ```js
    describe("my class", () => {
    	fit("does the work", () => {
    		expect(1 + 1).toBe(2);
    		expect(1 + 2).toBe(3);
    		expect(1 + 3).toBe(4);
    		expect(1 + 4).toBe(5);
    		expect(1 + 5).toBe(6);
    		expect(1 + 6).toBe(7);
    	});
    });
    ```
2. Enable `jasmine/no-focused-tests`
3. Run `npx eslint --format=json .`
4. The `endLine` is 9.

**Expected behaviour**

That `endLine` is the same as the `line`, i.e., 2.

The rule [mocha/no-exclusive-tests](https://github.com/lo1tuma/eslint-plugin-mocha/blob/master/lib/rules/no-exclusive-tests.js) does what I expect by only returning the `only` part as being wrong:

![2020-04-12-162140](https://user-images.githubusercontent.com/217543/79071047-b8e53d00-7cd9-11ea-9310-ad09162ca93d.png)

**Context**

- Node.js version: 10.17.0
- eslint-plugin-jasmine version:  4.1.0
- eslint version: 6.8.0
- Operating system: Fedora 31

**Comment by DamienCassou** [comment]
Please see #231 for a fix.

**Comment by tlvince** [comment]
Closed by 796dddf
